book/06-github/sections/2-contributing.asc

```diff
@@ -1,78 +1,78 @@
 
-.The ``Fork'' button.
-image::images/forkbutton.png[The ``Fork'' button.]
+.Ang ``Fork'' na pindutan.
+image::images/forkbutton.png[Ang ``Fork'' na pindutan.]
 
-After a few seconds, you'll be taken to your new project page, with your own writeable copy of the code.
+Pagkatapos ng ilang segundo, dadalhin ka sa pahina ng iyong bagong proyekto, na may sariling kopya ng code na maaaring mabago.
 
 
 [[_github_flow]]
-==== The GitHub Flow
+==== Ang Daloy ng GitHub
 
 (((GitHub, Flow)))
-GitHub is designed around a particular collaboration workflow, centered on Pull Requests.
-This flow works whether you're collaborating with a tightly-knit team in a single shared repository, or a globally-distributed company or network of strangers contributing to a project through dozens of forks.
-It is centered on the <<_git_branching#_topic_branch>> workflow covered in <<_git_branching#_git_branching>>.
+Ang GitHub ay dinisenyo sa paligid ng isang partikular na workflow sa pakikipagtulungan, nakasentro sa mga Kahilingan ng Pull.
+Ang daloy na ito ay gumagana kung nakikipagtulungan ka sa isang mahigpit na pangkat sa isang solong ibinahaging repositoryo, o isang kompanyang ibinahagi sa mundo o network ng mga estranghero na nag-aambag sa isang proyekto sa pamamagitan ng dose-dosenang mga fork.
+Ito ay nakasentro sa workflow na <<_git_branching#_topic_branch>> na tinalakay sa <<_git_branching#_git_branching>>.
 
-Here's how it generally works:
+Narito kung paano ito gumagana:
 
-1. Fork the project
-2. Create a topic branch from `master`.
-3. Make some commits to improve the project.
-4. Push this branch to your GitHub project.
-5. Open a Pull Request on GitHub.
-6. Discuss, and optionally continue committing.
-7. The project owner merges or closes the Pull Request.
+1. I-fork ang proyekto
+2. Lumikha ng isang branch ng paksa mula sa `master`.
+3. Gumawa ng ilang commits upang mapabuti ang proyekto.
+4. I-push ang branch na ito sa iyong proyekto sa GitHub.
+5. Magbukas ng isang Kahilingan ng Pull sa GitHub.
+6. Talakayin, at opsyonal na patuloy na gumawa.
+7. Pagsasamahin o isasara ng may-ari ng proyekto ang Kahilingan ng Pull.
 
-This is basically the Integration Manager workflow covered in <<_distributed_git#_integration_manager>>, but instead of using email to communicate and review changes, teams use GitHub's web based tools.
+Ito ang karaniwan na workflow ng Tagapamahala ng Paglagom na tinalakay sa <<_distributed_git#_integration_manager>>, ngunit sa halip na gagamit ng email sa pakikipag-ugnayan at pagsusuri ng mga pagbabago, ang pangkat ay gumagamit ng mga kagamitan ng GitHub na nakabatay sa web.
 
-Let's walk through an example of proposing a change to an open source project hosted on GitHub using this flow.
+Talakayin natin ang isang halimbawa ng pagpapanukala ng pagbabago sa isang open-source na proyekto na naka-host sa GitHub gamit ang daloy na ito.
 
-===== Creating a Pull Request
+===== Paglilikha ng isang Kahilingan ng Pull
```

Comment:
`Paglilikha ng isang Kahilingan ng Pull ` to `Paglikha ng isang Pull na Kahilingan`